pkg/estimator/client/service.go

```diff
@@ -0,0 +1,68 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	v1 "k8s.io/api/core/v1"
```

Comment:
using corev1 here

Reply:
Is there any code style file I can use to configure GoLand? Or any tool to do code style check? The goimports tools have different style then `corev1 `.

Reply:
Please check `hack/verify-import-aliases.sh` or `hack/.import-aliases`. corev1 is the alias for "k8s.io/api/core/v1"

Reply:
Eh, I found out `hack/verify-import-aliases.sh` `-include-path` doesn't include `pkg/estimator/client`, so `make verify` will not check go files under this path.

Reply:
Done